app/data/get-notification.js

```diff
@@ -4,7 +4,7 @@ const { serverConfig } = require('../config')
 const getNotification = async (id) => {
   try {
     const query = `query {
-      notification(id: ${id}) {
+      notification(id: "${id}") {
```

Comment:
Ahh, yeah was looking between the `get-notification.js` and the `get-notifications-by-sbi.js` files and didn't spot this difference, thanks!